joomla/joomla-cms
# Fix the calendars buttons

Pull Request for Issue https://github.com/joomla/joomla-cms/pull/15573  .

### Summary of Changes
Buttons are messed up if today button is not enabled


### Testing Instructions
Insert the following code to /administrator/templates/isis/index.php after line 288:
```html
		<?php echo JHtml::_('calendar', '', 'Some name', 'someid', '%Y-%m-%d', array('todayBtn' => false)); ?>
```
And then visit any page in the admin area (and check the calendar at the top)
Apply patch and refresh that page


### Expected result

Expected a nice calendar:
![screen shot 2017-04-27 at 11 55 16](https://cloud.githubusercontent.com/assets/3889375/25476050/8ea6610a-2b41-11e7-9963-e65409713f40.png)


### Actual result

Calendar should render correctly but its not:
![screen shot 2017-04-27 at 11 54 39](https://cloud.githubusercontent.com/assets/3889375/25475967/31996fc0-2b41-11e7-8879-24d406ec28b2.png)

### Documentation Changes Required
